duo_openvpn_as.py

```diff
@@ -557,6 +562,10 @@ def result(self):
     def status(self):
         return self.get('status')
 
+    @status.setter
+    def status(self, value):
+        self['status'] = value
+
```

Comment:
This does not appear to be used anywhere.

Reply:
ah good point. I just included it for completeness since the "alias" `msg` can set, I figured allowing one to set the `status` made sense.
Thanks for looking this over!
If you get a chance to pull these changes and test on your local OpenVPN server let me know. I was thinking of waiting to merge until I get the thumbs up from you. But let me know if you don't have capacity to run that test right now.

Reply:
When I saw the setters initially I was a bit confused because adding setters lets you modify an upstream response, which should technically be immutable.  Then I saw why you added the message setter because I missed a use-case in the code I modified, so I get it, but I sort of feel that changing the upstream response shouldn't be encouraged.

That said, I don't think it's a big deal.  👍🏽